modules/imgproc/src/color_lab.cpp

```diff
@@ -1585,6 +1585,93 @@ struct RGB2Lab_b
 
         i = 0;
 
+#if CV_NEON
+
+        // Define some scalar constants which we will make use of later
+        const int one_lsh_by_lab_shift_sub_one = (1 << ((lab_shift)-1));
+        const int one_lsh_by_lab_shift2_sub_one = (1 << ((lab_shift2)-1));
+        const int one_shift_lab_mul_128 = 128*(1 << lab_shift2);
+
+        // Convert 4 batches of 4 uint8_t RGBs, hence we stride in 16s
+        for(; i <= n - 16; i += 16, src += scn*16, dst += 3*16 )
```

Comment:
I think you might want to use `v_uint8::nlanes` here instead of 16?

Reply:
Done, also changed the types throughout to not hard code the number of lanes